<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <widget class="QFrame" name="f_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="SlidingStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="f_child">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QFrame" name="f_menu">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>55</y>
      <width>230</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_mixer">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/img_2.png</normaloff>:/img/img_2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_interpreter">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/img_1.png</normaloff>:/img/img_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>52</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MylTime" name="l_time">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MybtnSettings" name="btn_settings">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/img_3.png</normaloff>:/img/img_3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="StatsWidget" name="StatsWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1030</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="l_stat"/>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="l_stat_1"/>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="l_stat_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="VersionWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>4</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_1">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_2">
       <property name="text">
        <string>Copyright (c) 2022 | OSx</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="TimeWidget" name="TimeWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>260</width>
      <height>310</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Biennale Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="vL_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QFrame" name="container_1">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vL_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="l_time_1">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="l_date_1">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>220</height>
        </size>
       </property>
       <property name="gridVisible">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QCalendarWidget::NoSelection</enum>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::NoHorizontalHeader</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_hide">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>5</y>
      <width>550</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>\/</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <zorder>TimeWidget</zorder>
   <zorder>VersionWidget</zorder>
   <zorder>StatsWidget</zorder>
   <zorder>f_menu</zorder>
   <zorder>btn_hide</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatsWidget</class>
   <extends>QFrame</extends>
   <header>widgets.stats_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MybtnSettings</class>
   <extends>QPushButton</extends>
   <header>widgets.btn_settings.h</header>
  </customwidget>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>widgets.sliding_stacked_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeWidget</class>
   <extends>QFrame</extends>
   <header>widgets.time_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MylTime</class>
   <extends>QLabel</extends>
   <header>widgets.l_time.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
